docs/development/index.rst

```diff
@@ -0,0 +1,41 @@
+:orphan:
+
+***********************
+Contributing to Astropy
+***********************
+
+So you are interested in contributing to the Astropy Project?  Excellent! We
+love contributions! Astropy is open source, built on open source, and we'd
+love to have in our community.
```

Comment:
".. love to have *you* in our community." ?

Reply:
Thanks, fixed.